extensions/wikia/UserRenameTool/RenameUserProcess.class.php

```diff
@@ -1,78 +1,18 @@
 <?php
 
+use Wikia\DependencyInjection\Injector;
+
 class RenameUserProcess {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "RenameUserProcess" has 25 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)

[project-id]: wikia:app